### PR TITLE
Add Gemma Support and Conversion Scripts

### PR DESCRIPTION
**Pull request by yaoyu-33**
# What does this PR do ?

Add Gemma Support and Conversion Scripts

**Collection**: [NLP]

# Changelog 
- Add specific line by line info of high level changes in this PR.

# Usage
* You can potentially add a usage example below

```python
# Add a code snippet demonstrating how to use this 
```

# Jenkins CI
To run Jenkins, a NeMo User with write access must comment `jenkins` on the PR.

# Before your PR is "Ready for review"
**Pre checks**:
- [ ] Make sure you read and followed [Contributor guidelines](https://github.com/NVIDIA/NeMo/blob/main/CONTRIBUTING.md)
- [ ] Did you write any new necessary tests?
- [ ] Did you add or update any necessary documentation?
- [ ] Does the PR affect components that are optional to install? (Ex: Numba, Pynini, Apex etc)
  - [ ] Reviewer: Does the PR have correct import guards for all optional libraries?
  
**PR Type**:
- [ ] New Feature
- [ ] Bugfix
- [ ] Documentation

If you haven't finished some of the above items you can still open "Draft" PR.

## Who can review?

Anyone in the community is free to review the PR once the checks have passed. 
[Contributor guidelines](https://github.com/NVIDIA/NeMo/blob/main/CONTRIBUTING.md) contains specific people who can review PRs to various areas.

# Additional Information
* Related to # (issue)

### CODE REVIEW

**Comment by yaoyu-33** [comment]
jenkins

**Review by cuichenx — CHANGES_REQUESTED** [review]
could you update the config file?

**Review by cuichenx — APPROVED** [review]
LGTM!

**Comment by yaoyu-33** [comment]
jenkins

File: nemo/collections/nlp/models/language_modeling/megatron_gpt_model.py [hunk 1]
```diff
@@ -140,6 +141,12 @@ def get_specs(spec_name, num_experts=None):
         raise ValueError(f"Spec name '{spec_name}' is not recognized.")
 
 
+class EmbeddingScalingMixin(torch.nn.Module):
+    def forward(self, **kwargs):
+        embeddings = super().forward(**kwargs)
+        return embeddings * (self.config.hidden_size ** 0.5)
```
[the remark this answers is not in the document]
Comment (ericharper) [reply nemo/collections/nlp/models/language_modeling/megatron_gpt_model.py:147]: also could you add a docstring explaining why we have this class?

[the remark this answers is not in the document]
Comment (yaoyu-33) [reply nemo/collections/nlp/models/language_modeling/megatron_gpt_model.py:147]: Yes, I think only 0.5 makes sense for now. It means sqrt(hidden_dim)
I will add a docstring to it!

**Comment by yaoyu-33** [comment]
jenkins

**Comment by yaoyu-33** [comment]
jenkins

**Review by cuichenx — APPROVED** [review]

**Comment by yaoyu-33** [comment]
jenkins

**Comment by yaoyu-33** [comment]
jenkins

**Pull request merged**